### ISSUE TITLE
Trigger Suggest (⌃Space) does nothing in "Interactive Playground"

### ISSUE THREAD

**Issue by GandalfSaxe** [Microsoft/vscode#59066]
Issue Type: <b>Bug</b>

When I put the cursor in front of the error underline and press ⌃Space in the "Interactive Playground" demo, nothing happens. This is without any extensions:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/12127864/45846973-a9825580-bd29-11e8-9aaf-608d9b196b4c.png">

Why is it not working?

VS Code version: Code - Insiders 1.28.0-insider (af3614909b837c46a260666cb7baab7a8468a9c8, 2018-09-20T14:01:25.124Z)
OS version: Darwin x64 18.0.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz (8 x 2500)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: unavailable_software<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 4, 4|
|Memory (System)|16.00GB (0.05GB free)|
|Process Argv|/Users/gandalf/Documents/portable/Visual Studio Code - Insiders.app/Contents/MacOS/Electron -psn_0_427415428|
|Screen Reader|no|
|VM|0%|

</details>Extensions: none

**Comment by chrmarti** [comment]
Does this reproduce with an empty window (Cmd + Shift + N)?

**Comment by GandalfSaxe** [comment]
Yes, I all extensions disabled, and `Interactive Playground` as the only tab, I still get nothing when I press `⌃Space` on any cursor position around ``${req.}`);``:

![image](https://user-images.githubusercontent.com/12127864/45980118-7275b200-c051-11e8-8c34-4d1c9d98eec6.png)

**Comment by jarrodldavis** [comment]
I had this same issue on macOS Mojave (not sure if this was an issue with previous macOS versions) with VS Code Stable and I fixed it by disabling the global shortcuts in System Preferences > Keyboard > Shortcuts > Input Sources. I think macOS may have altered Input Sources settings because I also had menubar-related settings toggled on when I didn't explicitly enable them.

<img width="780" alt="screen shot 2018-10-07 at 12 59 04 pm" src="https://user-images.githubusercontent.com/235875/46585675-de342380-ca30-11e8-8f43-100e4e3ab63c.png">

If you don't see Input Sources under Keyboard > Shortcuts, try adding an additional Input Source in Keyboard > Input Sources.

<img width="780" alt="screen shot 2018-10-07 at 1 01 31 pm" src="https://user-images.githubusercontent.com/235875/46585698-26534600-ca31-11e8-9231-f4dcb3d1a5f2.png">

After disabling the Input Source keyboard shortcuts, I had to quit and restart VS Code to get Trigger Suggest to work again.

**Comment by GandalfSaxe** [comment]
@jarrodldavis Can confirm it works, and fixed the issue.

The VSCode team should probably think about this since it's a system default, so it won't work for the vast majority of macOS users. I don't mind turning off the shortcut to cycle input (never use it), but it's good to know that it has to be done :)

**Comment by GandalfSaxe** [comment]
@bpasero True, but you have still chosen to use a shortcut that is already a system default shortcut on all bi-lingual systems, which is not an insignificant amount among developers, and it will silently fail on 99% of these systems.

I would suggest as a minimum you add some text to the interactive tutorial that informs the user of how to make ⌃Space work :)

**Comment by bpasero** [comment]
@GandalfSaxe this keybinding is pretty much an established keybinding for any editor to trigger completion items, so I would not say that we "chose" this keybinding, we just follow standards. I leave it up to @chrmarti to decide if something should change.

Is this specific to the playground? I would expect this keybinding to fail in any context, even in the normal editor?

**Comment by GandalfSaxe** [comment]
@bpasero you’re that that’s it’s pretty conventional. Oddly enough I don’t remember any issue here I was using PyCharm. I’m not sure though.

Correct, it didn’t work anywhere in VSCode for me, I just noticed it first in the interactive playground.

**Comment by chrmarti** [comment]
@bpasero Should have said: FYI, since you're on Mojave (IIRC).

I wonder how other editors deal with this since it's such a common keybinding.

**Comment by bpasero** [comment]
I actually think this keybinding has always been there, not just with Mojave.
